theme/templates/base.html

```diff
@@ -28,11 +28,15 @@
 <link rel="stylesheet" type="text/css" href="{% static 'css/identifiers.css' %}">
 <link rel="stylesheet" type="text/css" href="{% static 'css/hydroshare_extends.css' %}">
 
+
 {% block extra_css %}{% endblock %}
 {% endcompress %}
 
 {% compress js %}
-<script type="text/javascript" src="{% static 'mezzanine/js/'|add:settings.JQUERY_FILENAME %}"></script>
+{# Pabitra: commenting the use of mezzanine supplied jquery js file as it's using a newer version which breaks lot of custom js code #}
```

Comment:
comments?

Reply:
Yeah it was intentional for the purpose of review as I am not sure if not using jquery provided by Mezzanine will break UI somewhere. Do you anticipate any issue with not using jquery from Mezzanine? If no issue then I will remove the comment and commented code.

Reply:
got it, I understand now, thanks. No I don't think this should create any issues. But we do need to swap it in a few other places:
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/hs_core/templates/500.html#L39
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/hs_core/templates/404.html#L39
https://github.com/hydroshare/hydroshare/blob/4738-django-3_2/hs_core/templates/503.html#L39

Reply:
Fixed these.